src/app/researcher/study/request/[memberIdentifier]/study-proposal.tsx

```diff
@@ -27,100 +39,113 @@ export const StudyProposalForm: FC<{
     const fileUpload = getFileUploadIcon(color, studyProposalForm.values.descriptionDocument?.name ?? '')
     const irbFileUpload = getFileUploadIcon(color, studyProposalForm.values.irbDocument?.name ?? '')
     const agreementFileUpload = getFileUploadIcon(color, studyProposalForm.values.agreementDocument?.name ?? '')
+
     const { user } = useUser()
 
     return (
         <Paper p="md">
-            <Title order={4} c="purple">
-                Study Proposal
-            </Title>
+            <Title order={4}>Study Proposal</Title>
             <Divider my="sm" mt="sm" mb="md" />
-            <Text>
-                This section is key to your proposal, as it defines the analysis that will generate the results you’re
-                intending to obtain from the Member’s data. Upload any necessary files to support your analysis. In this
-                iteration, we currently support .r and .rmd files.
+            <Text mb="md">
```

Comment:
i typically trend towards using a `<Stack>` component with the gap to handle spacing in a more "flex layout" approach. ie all these sibling items could have an even gap vs setting margins on them one by one